playbooks/roles/dev-patcher/tasks/patch-repos.yml

```diff
@@ -9,29 +9,27 @@
 
 - name: Get patch {{ patchnumber }} details
   uri:
-    url: "{{ dev_patcher_baseurl }}/?q=is:open+{{ patchnumber }}&o=ALL_REVISIONS"
+    url: "{{ dev_patcher_baseurl }}/?q=is:open+{{ patchnumber }}&q=is:merged+{{ patchnumber }}&o=ALL_REVISIONS"
```

Comment:
(nit) could drop both these query parts
`url: "{{ dev_patcher_baseurl }}/?q={{ patchnumber }}&o=ALL_REVISIONS"

Reply:
Oh. And I thought I tested it first :-)